Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/index.md

```diff
@@ -3,13 +3,13 @@ versionFrom: 8.0.0
 versionTo: 9.0.0
 ---
 
-# Creating, Saving, and Publishing Content
+# Creating, Saving and Publishing, Comparing, and Transferring Content
```

Comment:
The title becomes a little cluttered. Can it somehow be made a little more simple?

Reply:
Changed it to Creating, Saving and Publishing Content Options